_posts/2013-04-08-generators-and-blueprints.md

```diff
@@ -195,6 +195,16 @@ used in pod structure. If the blueprint does not require pod
 support, simply use the blueprint name instead of the
 `__path__` token.
 
+The __`__root__`__ token is substituted with either `app` or
```

Comment:
Yeah, it's emphasized inline code. [It looks okay after it's processed](http://www.ember-cli.com/#files).